analysis-module/src/main.py

```diff
@@ -9,10 +9,12 @@
 from src import prometheus
 from src import analysis_logger
 from src import plotting
+from src import analysis_cproc
+from src import config
```

Comment:
This import is not being used. Aand acually `config.py` is not necessary since we already have `load_json` which basically does the same. If you don't like it in `topology.py` we can move it to another file, but can have just one funcion.

Also, we are loading twice the configuration, in `config` and `simulation_config`.